### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by MaximKudriavtsev**

### CODE REVIEW

File: packages/dx-react-chart-bootstrap4/package.json [hunk 1]
```diff
@@ -62,31 +62,31 @@
...
+    "classnames": "^2.2.6",
+    "prop-types": "^15.6.2"
   },
   "peerDependencies": {
     "@devexpress/dx-chart-core": "1.3.0",
     "@devexpress/dx-react-chart": "1.3.0",
     "@devexpress/dx-react-core": "1.3.0",
     "react": "^16.2.0",
-    "reactstrap": "6.0.1"
+    "reactstrap": "6.1.0"
```
Comment (kvet) [inline packages/dx-react-chart-bootstrap4/package.json]: Hm, why we don't use open range here?

File: packages/dx-react-chart-material-ui/package.json [hunk 2]
```diff
@@ -60,32 +60,32 @@
...
-    "prop-types": "^15.6.1"
+    "classnames": "^2.2.6",
+    "prop-types": "^15.6.2"
   },
   "peerDependencies": {
     "@devexpress/dx-chart-core": "1.3.0",
     "@devexpress/dx-react-chart": "1.3.0",
     "@devexpress/dx-react-core": "1.3.0",
-    "@material-ui/core": "^1.1.0",
+    "@material-ui/core": "^1.2.3",
```
Comment (kvet) [inline packages/dx-react-chart-material-ui/package.json]: Is it needed to update minimal required version of mui?

File: packages/dx-react-grid-material-ui/package.json [hunk 3]
```diff
@@ -62,32 +62,32 @@
...
-    "prop-types": "^15.6.1"
+    "classnames": "^2.2.6",
+    "prop-types": "^15.6.2"
   },
   "peerDependencies": {
     "@devexpress/dx-grid-core": "1.3.0",
     "@devexpress/dx-react-core": "1.3.0",
     "@devexpress/dx-react-grid": "1.3.0",
-    "@material-ui/core": "^1.1.0",
+    "@material-ui/core": "^1.2.3",
```
Comment (kvet) [inline packages/dx-react-grid-material-ui/package.json]: The same place

File: packages/dx-react-chart-material-ui/package.json (hunk 2 above)
Comment (oliviertassinari) [reply packages/dx-react-chart-material-ui/package.json]: Probably not

**Pull request merged**